### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by DefaultDaddy**
The code has been refactored a bit (To follow the same naming schemes). Additonally, there has been created unit tests, that has 100% test coverage for CheepService and CheepRepository.

### CODE REVIEW

**Review by MrBStones — APPROVED** [review]
Lgtm

File: test/Chirp.Razor.Tests/ChirpTests.cs [hunk 1]
```diff
@@ -210,7 +190,7 @@ public void TestGetCheepsFromAuthorWithID(string authorId)
         // Arrange
         
         // Act
-        var result = _cheepRepo.ReadCheepsByID(authorId);
+        var result = _cheepService.GetCheepsFromAuthorByID(authorId);
         // Assert
         Assert.NotEmpty(result);
```
Comment (JKlarlund) [inline test/Chirp.Razor.Tests/ChirpTests.cs:195]: Maybe we need to change assert to check if the result is actually written by the author

File: test/Chirp.Razor.Tests/ChirpTests.cs [hunk 2]
```diff
@@ -222,7 +202,7 @@ public void TestGetCheepsFromAuthorWithName(string name)
         // Arrange
         
         // Act
-        var result = _cheepRepo.ReadCheepsByName(name);
+        var result = _cheepService.GetCheepsFromAuthorByName(name);
         // Assert
         Assert.NotEmpty(result);
```
Comment (JKlarlund) [inline test/Chirp.Razor.Tests/ChirpTests.cs:207]: Same here

File: test/Chirp.Razor.Tests/ChirpTests.cs [hunk 3]
```diff
@@ -232,7 +212,7 @@ public void TestGetCheepsFromAuthorWithName(string name)
     public void TestGetAuthorWithId(string id, string userName)
     {
         //Act
-        var result = _cheepRepo.GetAuthor(id);
+        var result = _cheepService.GetAuthorByID(id);
         
         //Assert
         Assert.NotNull(result);
```
Comment (JKlarlund) [inline test/Chirp.Razor.Tests/ChirpTests.cs:218]: And here

**Review by JKlarlund — CHANGES_REQUESTED** [review]
Nice work!

**Review by JKlarlund — APPROVED** [review]
Good work!

**Pull request merged**